_data/retail.yml

```diff
@@ -72,6 +72,12 @@ websites:
       img: groupon.png
       tfa: No
 
+    - name: Home Depot
+      url: http://www.homedepot.com
```

Comment:
No, it's because it's just using their CDN's default stuff. The CN listed on the cert is `a248.e.akamai.net`. In other words, their home page is not properly configured for HTTPS.

Their secure pages are actually on a separate subdomain that is properly configured, `secure2.homedepot.com`. Even then they use mixed content so the URL bar isn't green.